football_now.rb

```diff
@@ -11,50 +12,69 @@ def start
             response = HTTParty.get(BASE_URL)
             games = response.parsed_response['rss']['channel']['item']
 
-            games.each do |game|
-                puts_game(game)
-            end
+            city_prompt
+
+            print_games(games)
 
         rescue SocketError => e
             puts "Check your internet connection. Error message: #{e.message}".light_red.bold
         end
 
         private
-        
-        def puts_game(game)
-            game_splitted = game['title'].split(' - ')
-            puts %Q(#{ datetime_adjust(game['pubDate']) } #{Paint[game_splitted.last, :white, :italic]} - #{ Paint[colorize_benfica(game_splitted.first), :bold] }\n)
-        end
 
-        def datetime_adjust(date)
-            date_splitted = date.split(' ').drop(1)
+        def city_prompt
+            puts "\nPORTUGUESE SPORTV TV CHANNELS"
+            puts "\nEnter the city to convert games date/times into"
+            puts "\nNOTE: type #{Paint['help', :green]} for the list of cities/timezones\n"
 
-            new_time = offset(date_splitted, 'Berlin')
-            date = date_adjust(date_splitted, new_time)
+            user_input
+        end
 
-            "#{date} #{new_time}"
+        def user_input
+            input = gets.chomp
+            input.downcase == 'help' ? cities_tz_list : @city_tz = find_tzinfo(input.capitalize)
         end
 
-        def offset(date_splitted, city_tz)
-            Time.zone = 'London'
-            game_bst_time = Time.zone.parse("#{date_splitted[2]}-#{date_splitted[1]}-#{date_splitted.first} #{date_splitted.last}")
+        def cities_tz_list
+            ActiveSupport::TimeZone.all.sort_by { |o| o.name }.map {|o| puts o.name} 
+            city_prompt
```

Comment:
Hmmm this seems unorthodox to me.
Usually I can see loops with prompts - this is kinda loop - but it's not obvious.
Wouldn't it be easier to read if there was something like:
```
do
input = gets.chomp
input.downcase == 'help' ? cities_tz_list : continue
tzinfo = ActiveSupport::TimeZone.find_tzinfo(input)
while(not tzinfo)
```
loop in user input?
This would be more pythonic, but instead of tzinfo = ActiveSupport::TimeZone.find_tzinfo(input) I'd actually firstly go for verification of input, and if it's not correct, then just continue.

I don't know how to handle such things in ruby - maybe it's right way to do it, and I'm just not aware of it ; p